### DISCUSSION TITLE
Error handling for the spawn actor

### DISCUSSION THREAD

**Discussion by rcl-official** [statelyai/xstate#4874] [General]
I found that i am not able to handle the error from the spawn actor just like the invoke actor. And i did search the documents found nothing talk about the error handling for the spawn children.

**Comment by davidkpiano** [comment]
Can you share some code that shows that this is not working?

**Reply by Andarist** [reply]
I suspect the issue is about the fact that you can't easily attach an `onError` transition handler to your spawned actors (but you can do that with invoked ones)
